force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -71,7 +84,20 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Positive", "Negative", or "Zero", or null if the number is null.
      */
     public static String checkNumber(Integer a) {
-        return null; // Replace null with the variable you used to store the result
+        string result;
```

Comment:
In Apex, `String` is a class and a Type, so it is capitalized. Just something to note.